discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/CreateEnvironmentOptions.java

```diff
@@ -20,17 +20,41 @@
  */
 public class CreateEnvironmentOptions extends GenericModel {
 
+  /**
+   * Size of the environment.
+   */
+  public interface Size {
+    /** XS. */
+    String XS = "XS";
+    /** S. */
+    String S = "S";
+    /** MS. */
+    String MS = "MS";
+    /** M. */
+    String M = "M";
+    /** ML. */
+    String ML = "ML";
+    /** L. */
+    String L = "L";
+    /** XL. */
+    String XL = "XL";
+    /** XXL. */
+    String XXL = "XXL";
+    /** XXXL. */
+    String XXXL = "XXXL";
+  }
+
   private String name;
   private String description;
-  private Long size;
+  private String size;
```

Comment:
I missed the Friday meeting, so I may have heard wrong, but wasn't the decision to leave `Size` in the SDK as a number for compatibility and add a `SizeString` parameter to be type `String`?

Reply:
You might be right @dpopp07, although then that'll involve us having more handwritten methods for backwards-compatibility in the next major release when the `Long` size is the only one left.

Reply:
This is for the request, not the response, right? I know it's been deprecated for a long time, so users shouldn't be including it in their request, but if they are it would probably break them. If you think this is okay and that we shouldn't be causing any issues, I'd prefer to do the same thing in the Node SDK and just change the type. So let me know!

Reply:
You can take the same approach as the one logan is taking here.

Reply:
@lpatino10 I agree, which is why I was I wasn't thrilled about doing it haha
@germanattanasio Sounds good, I'll make that change then